src/akeneo/harness/scripts/enable.sh.twig

```diff
@@ -52,6 +53,7 @@ static()
     ws app build
 
     passthru "docker-compose config --services | grep -v job-queue-consumer | xargs docker-compose up -d"
+    passthru sleep 60s # ensure mysql has had time to start
```

Comment:
can we change this to do tcp retry checks instead? having to always wait 60 seconds for mysql to be started is very inefficient. I can accept it as a separate PR tho when someone has some time

Reply:
this is no longer need so I will remove it